src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -36,7 +36,7 @@ const customHTMLElementModels = {
     }),
 };
 
-const defaultViewProps = {style: {alignItems: 'flex-start'}};
+const defaultViewProps = {style: {alignItems: 'flex-start', userSelect: 'text'}};
```

Comment:
We could refactor this to use the newly added style, instead of hardcoding here.

Reply:
Possibly refactor both , use `Styles.alignItemsStart` instead of `alignItems: 'flex-start'`